### DISCUSSION TITLE
请求退款

### DISCUSSION THREAD

**Discussion by mcutdgreat** [getlantern/lantern-forum-cn#770] [Q&A]
昨日发现今年一月份因为网络卡顿，误操作连续两次购买了2年的专业版服务，之前专业版服务到期日为2026年，现在专业版到期日为2030年，因目前还未进入误操作付款的服务期间，请求退还这两次购买所需的金额 非常感谢。联系方式695101151@qq.com

**Comment by MaxMKMax** [comment]
@mcutdgreat 
您好. 请您提供我那两次的支付单据. 我帮您提交问题. 谢谢.

**Reply by mcutdgreat** [reply]
你好 请问退款了吗

**Comment by mcutdgreat** [comment]
好的，非常感谢回复，辛苦了



---原始邮件---
发件人: "Mike ***@***.***&gt;
发送时间: 2024年8月10日(周六) 凌晨0:35
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [getlantern/lantern-forum-cn] 请求退款 (Discussion #770)




 
@mcutdgreat
 您好. 请您提供我那两次的支付单据. 我帮您提交问题. 谢谢.

**Reply by MaxMKMax** [reply]
@mcutdgreat 
您好. 不客气呢. 又像您这样的用户是我们的荣幸. 非常感谢您支持蓝灯! 非常感谢您! 

同时请您记得提供我那两次错误付款的单据. 在这个帖子给我. 我才能帮您提交问题上去. 谢谢!

**Reply by mcutdgreat** [reply]
![Image_1723294861626](https://github.com/user-attachments/assets/5317ece1-feb0-49da-a8ac-72a7ea58f9d7)

**Reply by mcutdgreat** [reply]
请问还没有退款吗？在进行中吗？

**Reply by mcutdgreat** [reply]
> @mcutdgreat 您好. 不客气呢. 又像您这样的用户是我们的荣幸. 非常感谢您支持蓝灯! 非常感谢您!
> 
> 同时请您记得提供我那两次错误付款的单据. 在这个帖子给我. 我才能帮您提交问题上去. 谢谢!

请问还没走完程序吗

**Comment by mcutdgreat** [comment]
发给你了
![Image_1723294861626](https://github.com/user-attachments/assets/ec697d11-bf54-47b0-9ef1-a25699ab93a1)

**Comment by mcutdgreat** [comment]
已经两周了 请问还没有消息吗

**Comment by MaxMKMax** [comment]
@mcutdgreat 我帮您问一下. 谢谢您的耐心等待.

**Reply by mcutdgreat** [reply]
@Mike777Khoo
你好 几周前退款了一次购买的金额。
还有另外一次退款没有退， 非常感谢

**Comment by mcutdgreat** [comment]
@Mike777Khoo 
你好  几周前退款了一次购买的金额。
还有另外一次退款没有退， 非常感谢。

**Reply by vincentye9682** [reply]
@mcutdgreat 申请退款后多久退的款？

**Reply by mcutdgreat** [reply]
> @mcutdgreat 申请退款后多久退的款？

大概要一两个月，但我第二笔到现在还没退

**Reply by vincentye9682** [reply]
只要能退就还好，申请退款就不能使用是吧？不然不退？

**Reply by mcutdgreat** [reply]
恩

**Comment by MaxMKMax** [comment]
@mcutdgreat 不客气. 好的, 我再帮您问看支付部门. 请您耐心等待. 谢谢啦

**Reply by mcutdgreat** [reply]
请问第二笔退款何时还 已经好几个月了..@Mike777Khoo

**Reply by MaxMKMax** [reply]
@mcutdgreat 您好. 不好意思啊. 抱歉啊. 前几天我生病了. 不在线. 我来帮您问问看.

**Reply by mcutdgreat** [reply]
> @mcutdgreat 您好. 不好意思啊. 抱歉啊. 前几天我生病了. 不在线. 我来帮您问问看.

现在还有消息吗 第二笔还没收到。

**Comment by MaxMKMax** [comment]
@mcutdgreat 您好. 帮您提交问题上去了. 我让支付部门帮您查看.

**Reply by mcutdgreat** [reply]
第一笔还的很快，第二笔现在还没 希望跟进..
